Marlin/src/HAL/STM32/HAL.h

```diff
@@ -117,6 +117,14 @@
   #endif
 #endif
 
+#ifdef RS485_SERIAL_PORT
+  #if WITHIN(RS485_SERIAL_PORT, 1, 6)
```

Comment:
Now that we support 9 UARTs (https://github.com/MarlinFirmware/Marlin/commit/abd1896ce2cd37a184fc1a237482b2b0a3c16ae2), should this be bumped?